components/dixa/actions/add-message/add-message.mjs

```diff
@@ -0,0 +1,67 @@
+import dixa from "../../dixa.app.mjs";
+
+export default {
+  key: "dixa-add-message",
+  name: "Add Message to Conversation",
+  description: "Adds a message to an existing conversation. [See the documentation](https://docs.dixa.io/openapi/dixa-api/v1/tag/Conversations/#tag/Conversations/operation/postConversationsConversationidMessages).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    dixa,
+    endUserId: {
+      propDefinition: [
+        dixa,
+        "endUserId",
+      ],
+    },
+    conversationId: {
+      propDefinition: [
+        dixa,
+        "conversationId",
+        ({ endUserId }) => ({
+          endUserId,
+        }),
+      ],
+    },
+    content: {
+      type: "string",
+      label: "Content",
+      description: "Content of the message",
+    },
+    direction: {
+      propDefinition: [
+        dixa,
+        "direction",
+      ],
+      reloadProps: true,
+    },
+    agentId: {
+      propDefinition: [
+        dixa,
+        "agentId",
+      ],
+    },
+  },
+  async additionalProps(props) {
+    props.agentId.hidden = !(this.direction === "Outbound");
```

Comment:
I think it'd be easier to just use the "not equal" operator in cases such as this
```suggestion
    props.agentId.hidden = this.direction !== "Outbound";
```